ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/util/BlockBlobSidecarsTrackersPoolImplTest.java

```diff
@@ -220,16 +225,58 @@ public void onNewBlobSidecar_shouldPublishWhenOriginIsLocalEL() {
             .signedBeaconBlockHeader(dataStructureUtil.randomSignedBeaconBlockHeader(currentSlot))
             .build();
 
+    when(blobSidecarGossipValidator.markForEquivocation(blobSidecar1)).thenReturn(true);
+
     blockBlobSidecarsTrackersPool.onNewBlobSidecar(blobSidecar1, RemoteOrigin.LOCAL_EL);
     blockBlobSidecarsTrackersPool.onNewBlobSidecar(blobSidecar2, RemoteOrigin.GOSSIP);
     blockBlobSidecarsTrackersPool.onNewBlobSidecar(blobSidecar3, RemoteOrigin.RPC);
 
     assertBlobSidecarsCount(3);
     assertBlobSidecarsTrackersCount(3);
 
+    verify(blobSidecarGossipValidator).markForEquivocation(blobSidecar1);
     verify(blobSidecarPublisher, times(1)).accept(blobSidecar1);
   }
 
+  @Test
+  public void onNewBlobSidecar_shouldPublishEvenWhenOriginIsLocalELButEquivocating() {
```

Comment:
I'd rename the test to `onNewBlobSidecar_shouldPublishWhenOriginIsLocalELAndNotEquivocating`